python-package/mlbox/preprocessing/reader.py

```diff
@@ -306,45 +326,59 @@ def train_test_split(self, Lpath, target_name):
 
         if (type(Lpath) != list):
 
-            raise ValueError("You must specify a list of paths to load all the data")
+            raise ValueError("You must specify a list of paths "
+                             "to load all the data")
 
         elif (self.to_path is None):
 
-            raise ValueError("You must specify a path to save your data and make sure your files are not already saved")
+            raise ValueError("You must specify a path to save your data "
+                             "and make sure your files are not already saved")
 
         else:
 
-            ######################################################
-            ################## reading the files ##################
-            ######################################################
+            ##############################################################
+            #                    Reading the files
+            ##############################################################
 
             for path in Lpath:
 
-                ### reading each file ###
-                df = self.clean(path, date_strategy="complete", drop_duplicate=False)
+                # Reading each file
+
+                df = self.clean(path,
+                                date_strategy="complete",
+                                drop_duplicate=False)
+
+                # Checking if the target exists to split into test and train
 
-                ### checking if the target exists to split into test and train
                 if (target_name in df.columns):
 
-                    df_train[path] = df[df[target_name].isnull() == False].drop(target_name, axis=1)
-                    df_test[path] = df[df[target_name].isnull() == True].drop(target_name, axis=1)
-                    y_train[path] = df[target_name][df[target_name].isnull() == False]
+                    is_null = df[target_name].isnull()
+
+                    df_train[path] = df[~is_null].drop(target_name, axis=1)
+                    df_test[path] = df[is_null].drop(target_name, axis=1)
+                    y_train[path] = df[target_name][is_null]
 
                 else:
 
                     df_test[path] = df
 
             del df
 
-            # exceptions
-            if (sum([df_train[path].shape[0] for path in df_train.keys()]) == 0):
-                raise ValueError("You have no train dataset. Please check that the target name is correct")
+            # Exceptions
+
+            if (sum([df_train[path].shape[0]
+                     for path in df_train.keys()]) == 0):
+                raise ValueError("You have no train dataset. "
+                                 "Please check that the "
+                                 "target name is correct.")
 
-            if ((sum([df_test[path].shape[0] for path in df_test.keys()]) == 0) & (self.verbose)):
+            if ((sum([df_test[path].shape[0]
+                      for path in df_test.keys()]) == 0) & (self.verbose)):
```

Comment:
I'm not a fan of using bit operators for `or` and `and`. While the results may be the same, I think we are losing readability and misusing the operators